docker-compose.yml

```diff
@@ -17,9 +17,8 @@ services:
     ports:
       - "9092:9092"
     environment:
-      KAFKA_ADVERTISED_LISTENERS: LISTENER_DOCKER_INTERNAL://kafka1:19092,LISTENER_DOCKER_EXTERNAL://${DOCKER_HOST_IP:-127.0.0.1}:9092
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: LISTENER_DOCKER_INTERNAL:PLAINTEXT,LISTENER_DOCKER_EXTERNAL:PLAINTEXT
-      KAFKA_INTER_BROKER_LISTENER_NAME: LISTENER_DOCKER_INTERNAL
+      KAFKA_BROKER_ID: 1
```

Comment:
why are you changing those? 
These values are actually from my project https://github.com/simplesteph/kafka-stack-docker-compose and work for most users although configuration might look a bit more complicated. 
I'd rather revert

Reply:
These lines blew up when I tried to run them with the latest Docker and Docker-compose.  Happy to revert though.